com.unity.formats.fbx/Tests/FbxTests/ModelExporterTest.cs

```diff
@@ -541,7 +541,21 @@ private void CompareMeshComponentAttributes(Mesh mesh, Mesh fbxMesh)
             Assert.AreEqual (mesh.tangents, fbxMesh.tangents);
         }
 
-        private string ExportSkinnedMesh(string fileToExport, out SkinnedMeshRenderer originalSkinnedMesh, out SkinnedMeshRenderer exportedSkinnedMesh){
+        private delegate void SetImportSettings(ModelImporter importer);
+        private string ExportSkinnedMesh(
+            string fileToExport, 
+            out SkinnedMeshRenderer originalSkinnedMesh,
```

Comment:
FYI: If 2018.3 has C#7+ support, you can return anonymous tuples: https://docs.microsoft.com/en-us/dotnet/csharp/tuples